students/carlos_novoa/lesson08/circle_class.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import math
+
+"""
+Lesson8 - Circle Class
+"""
+
+
+class Circle(object):
+    version = '0.1'
+
+    def __init__(self, radius=None, diameter=None):
```

Comment:
Note that the assignment only asks for a single required radius parameter for the constructor method. This should create a `radius` instance attribute (no getter and setter required)